app/Http/Controllers/Member/TaskController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers\Member;
+
+use App\Http\Controllers\Controller;
+use App\Models\Member;
+use App\Models\Task;
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class TaskController extends Controller
+{
+    /**
+     * Show Select Project to view tasks.
+     */
+    public function index()
+    {
+        $taskByProject = auth()->user()->projects()->paginate(config('app.pagination'));
+        return view('members.tasks.index', ['data' => $taskByProject, 'id' => Member::ID]);
+    }
+
+    /**
+     * Show Tasks from id project.
+     * @param $id
+     * @return Factory|View
+     */
+    public function show($id)
+    {
+        $listTask = auth()->user()->tasks()->where('project_id', $id)->paginate(config('app.pagination'));
+        $dataTask = [
+            'listTask' => $listTask,
+            'orderId' => Member::ID,
+            'project_id' => $id,
+            'paginate' => config('app.pagination')
+        ];
+        return view('members.tasks.tasks', ['dataTask' => $dataTask]);
```

Comment:
Với hàm này chỉ lấy được task theo 1 project nhất định à? Sao đặt là show nhỉ, show thì chỉ lấy 1 task để show ra thôi chứ ?

Reply:
Cái này ban đầu em sẽ cho member select project và sau đó hàm này sẽ show ra toàn bộ task theo project đấy ạ